src/styl/_components/_box.styl

```diff
@@ -11,10 +11,12 @@
 // Styleguide: Components.Box
 /:root
 /[data-theme=light]
+/[data-theme] [data-theme=light]
```

Comment:
I don't understand why this is usefull

Reply:
```html
<html data-theme="dark">
    <body>
        Dark content
        <div data-theme="light">
             Light content because specificity is stronger
        </div>
    </body>
```